### ISSUE TITLE
amnesia filter

### ISSUE THREAD

**Issue by DhruvaSambrani** [DhruvaSambrani/pentra#10]

**Comment by 20akshay00** [comment]
Depending on what exactly we want this to look like; it could be emulated by blit-ting an image texture over the entire window and animating the alpha value over the frames.

**Comment by DhruvaSambrani** [comment]
Can we apply a filter instead?

**Comment by DhruvaSambrani** [comment]
https://pyga.me/docs/ref/transform.html#pygame.transform.gaussian_blur

**Comment by 20akshay00** [comment]
Tried. Performance is horrible if you try to apply on full display surface.

**Comment by DhruvaSambrani** [comment]
Can you try box_blur also?

**Comment by 20akshay00** [comment]
Yeah, nope. Any kind of post processing on the full thing brings it down to 0 FPS. I had to try it with just the HUD to check if it even worked.

**Comment by 20akshay00** [comment]
Same. I'm not entirely sure if I'm doing it right though; 
`self._display_surf.blit(
            pygame.transform.box_blur(self._display_surf, 2), (0, 0)
        )`
gives me really low FPS.
` self._display_surf = pygame.transform.box_blur(self._display_surf, 2)
`
just freezes the display entirely.

**Comment by DhruvaSambrani** [comment]
`pygame.transform.box_blur(self._display_surf, 2, self._display_surf)`
